packages/create-discord-bot/src/index.ts

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+// eslint-disable-next-line n/shebang
+import { execSync } from 'node:child_process';
+import { cpSync, existsSync, mkdirSync } from 'node:fs';
+import path from 'node:path';
+import process from 'node:process';
+import { URL } from 'node:url';
+import chalk from 'chalk';
+import { program } from 'commander';
+
+// A directory must be specified.
+program
+	.description('Create a template bot.')
+	.argument('<directory>', 'The directory where this will be created.')
+	.parse();
```

Comment:
can u also add `name` arg and a `--ts` flag too?

Reply:
This should be done now.